akka-docs/src/main/paradox/typed/routers.md

```diff
@@ -43,8 +43,8 @@ Java
 The group router is created with a `ServiceKey` and uses the receptionist (see @ref:[Receptionist](actor-discovery.md#receptionist)) to discover
 available actors for that key and routes messages to one of the currently known registered actors for a key.
 
-Since the receptionist is used this means the group router is cluster aware out of the box and will pick up routees
-registered on any node in the cluster (there is currently no logic to avoid routing to unreachable nodes, see [#26355](https://github.com/akka/akka/issues/26355)).
+Since the receptionist is used this means the group router is cluster aware out of the box and will route to 
+registered actors on any node in the cluster that is reachable.
```

Comment:
No changes to the routers in this PR so there must be something missing. I think we said that they would prefer the reachable, but if that is empty it would fall back to all. Better to try to send to someone than just dropping.

Reply:
I didn't recall if we were 100% sure we wanted that so didn't go there yet.

Reply:
Will make sure this works first then add that on top